ci-operator/step-registry/openshift-qe/installer/bm/day2/sriov/openshift-qe-installer-bm-day2-sriov-workflow.yaml

```diff
@@ -0,0 +1,7 @@
+workflow:
```

Comment:
if you're not using the workflow in any of your tests now, I think this is just an extra file to keep up with and can remove